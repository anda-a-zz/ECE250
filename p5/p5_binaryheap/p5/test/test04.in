i Vancouver
i Vancouver
setd Vancouver;Seattle;12.89
setd Tokyo;Beijeng;13.42
setd Vancouver;Vancouver;11.1
i Edmonton
setd Vancouver;Edmonton;0
setd Vancouver;Edmonton;-1
s Calgary
degree Calgary
d Instanbul;Chicago
d Istanbul;Vancouver
d Vancouver;Vancouver
i Minnesota
setd Minnesota;Vancouver;1.2
d Minnesota;Edmonton
shortest_d Hamilton;Missisauga
shortest_d Hamilton;Hamilton
print_path Hamilton;Missisauga
print_path Hamilton;Missisauga
clear