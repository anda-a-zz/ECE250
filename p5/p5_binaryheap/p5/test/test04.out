success
failure
failure
failure
failure
success
failure
failure
not found
failure
failure
failure
failure
success
success
failure
failure
failure
failure
failure
success
